cli/loader_lib.c

```diff
@@ -31,6 +31,9 @@ void jl_loader_print_stderr3(const char * msg1, const char * msg2, const char *
 }
 
 /* Wrapper around dlopen(), with extra relative pathing thrown in*/
+/* If err, then loads the library successfully or panics.
+ * If !err, then loads the library or returns null if it could not be opened,
+ * or panics if opening failed for any other reason. */
```

Comment:
can you add a comment here for future readers about why we picked F_OK?

Reply:
Amended to elaborate.